sites/updates.thunderbird.net/thunderbird/128.0/dec24/index.html

```diff
@@ -13,61 +13,60 @@
 {% set fru_form_id = fru_form_id|default('20years') %}
 {% set utm_campaign = utm_campaign|default('dec24_appeal') %}
 {% set donation_base_url = donation_base_url|default(url('updates.appeal.dec24.donate')) %}
-{% set active_page = "128-appeal" %}
 
 {% extends "includes/base/base.html" %}
 {% from 'includes/macros/donate-button.html' import donate_button with context %}
 
 {% block base_css %}
-    <link href="/media/css/appeal-128.css" rel="stylesheet" type="text/css"/>
+  <link href="/media/css/appeal-128.css" rel="stylesheet" type="text/css"/>
 {% endblock %}
 {% block page_title %}{{ _('Thank you for choosing Thunderbird!') }}{% endblock %}
```

Comment:
Oh sorry, can we swap this to 
```suggestion
{% block page_title %}{{ _('Help Keep Thunderbird Alive!') }}{% endblock %}
```